templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/template_schema.json

```diff
@@ -17,7 +17,8 @@
         "description": "Select Windows image to use for VM",
         "enum": [
           "Windows 10",
-          "Server 2019 Data Science VM"
+          "Server 2019 Data Science VM",
+          "Custom Image From Gallery"
```

Comment:
Do we need this value? IIUC it will be visible on the UI

Reply:
Oops - good catch!